### PR TITLE
Rebrand to Healthsites

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: go-app.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ go.app = function() {
                     type: "SMS",
                     to_addr: contact.msisdn,
                     template: "Your nearest clinic is {{ results }}. Thanks " +
-                              "for using Clinic Finder"
+                              "for using Healthsites"
```
Comment (gsvr) [inline go-app.js:82]: This wasn't explicitly mentioned in the Jira ticket, but made sense to update.

**Comment by gsvr** [comment]
@imsickofmaps Note that I've only updated the message text, as per Jira ticket. The metric_store_name and Api URL's and so forth are still `clinicfinder`.

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**